.github/workflows/ci.yml

```diff
@@ -161,6 +161,33 @@ jobs:
           # build docs
           cargo doc --target ${{ matrix.target }} -p wgpu -p wgpu-core -p wgpu-info -p player --all-features --no-deps
 
+  wasm-test:
+    name: Test Wasm
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout repo
+        uses: actions/checkout@v3
+
+      - name: install rust stable
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+# doesn't work for this project because of https://github.com/rustwasm/wasm-pack/issues/1180
+#      - name: install wasm-pack
+#        uses: jetli/wasm-pack-action@v0.4.0
+#
+# install from fork until this is merged: https://github.com/rustwasm/wasm-pack/pull/1185
```

Comment:
```suggestion
					# doesn't work for this project because of https://github.com/rustwasm/wasm-pack/issues/1180
					#      - name: install wasm-pack
					#        uses: jetli/wasm-pack-action@v0.4.0
					#
					# install from fork until this is merged: https://github.com/rustwasm/wasm-pack/pull/1185
```